.github/workflows/master.yml

```diff
@@ -67,6 +72,7 @@ jobs:
         # It forces the sphinx version to 2.x.
         if [[ "$PYTHON_VERSION" < "3.6" ]]; then pip install "sphinx<3.0.0"; fi
         pip list
+        sudo apt install -y pandoc
```

Comment:
If the Sphinx build passes by installing `pandoc`, how about explicitly including it in `requirement-dev.txt` instead of `apt install` here?

Reply:
I believe pandoc has to be installed via `apt install`. The official instructions do not mention a way to install it via pip install
- https://pandoc.org/installing.html